### PR TITLE
Post merge

### PR DESCRIPTION
**Pull request by atarkowska**
# What this PR does

fix failing tests
cherry-picked to https://github.com/openmicroscopy/openmicroscopy/pull/5241

### CODE REVIEW

**Comment by atarkowska** [comment]
@jburel do you see any more name changes/fixes in m53?

**Comment by joshmoore** [comment]
Changes all make sense.  Are the tests currently being run somewhere?

**Comment by jburel** [comment]
@aleksandra-tarkowska sorry I missed the notification. I will have a look

**Comment by atarkowska** [comment]
@joshmoore https://10.0.51.150:8443

**Comment by joshmoore** [comment]
Thanks, @aleksandra-tarkowska. From my point-of-view, this is good to merge.

File: components/tools/OmeroPy/test/integration/metadata/test_populate.py [hunk 1]
```diff
@@ -133,10 +134,10 @@ def createDataset(self, names=("A1", "A2")):
         return ds.proxy()
 
     def createScreen(self, rowCount, colCount):
-        plate1 = self.test.importPlates(plateRows=rowCount,
-                                        plateCols=colCount)[0]
-        plate2 = self.test.importPlates(plateRows=rowCount,
-                                        plateCols=colCount)[0]
+        plate1 = self.test.import_plates(plateRows=rowCount,
```
Comment (jburel) [inline components/tools/OmeroPy/test/integration/metadata/test_populate.py]: This should be ``plate_rows`` and ``plate_cols`` if the method from testlib is used

**Comment by atarkowska** [comment]
@jburel thanks for the hint, looks like all metadata tests are passing now :)

**Comment by jburel** [comment]
It looks good now

**Comment by atarkowska** [comment]
cherry-picked fix from https://github.com/openmicroscopy/openmicroscopy/pull/5241 for py26 (just for consistency)

**Comment by joshmoore** [comment]
:+1:

**Pull request merged**